arangod/IResearch/IResearchAnalyzerFeature.cpp

```diff
@@ -2461,7 +2464,10 @@ Result IResearchAnalyzerFeature::loadAnalyzers(
       auto res = emplaceAnalyzer(result, analyzers, normalizedName, type, properties, features, revision);
 
       if (!res.ok()) {
-        return res; // caught error emplacing analyzer (abort further processing)
+        LOG_TOPIC("7cc7f", ERR, arangodb::iresearch::TOPIC)
+            << "analyzer '" << name
+            << "' ignored as emplace failed with reason:" << res.errorMessage();
+        return {};  // caught error emplacing analyzer  - skip it
```

Comment:
Is this change intended?
On the left you have the outer place the chance to see that something goes wrong here.
Now you log an error, but report Success.

Reply:
Yes. It`s a visitor for analyzers. It should return "Ok" or iteration will stop. So to skip it should still return OK.